packages/common/src/adapters/access.ts

```diff
@@ -34,3 +39,27 @@ export const accessConditionsFromSDK = (
     throw new Error(`Unsupported access gate type: ${input}`)
   }
 }
+export const accessConditionsToSDK = (
+  input: AccessConditions
+): TrackMetadata['downloadConditions'] => {
+  if ('follow_user_id' in input) {
+    return {
+      followUserId: encodeHashId(input.follow_user_id)
+    }
+  } else if ('nft_collection' in input) {
+    const collection = input.nft_collection as
+      | z.input<typeof EthCollectibleGatedConditions>
+      | z.input<typeof SolCollectibleGatedConditions>
+    return { nftCollection: collection }
+  } else if ('usdc_purchase' in input) {
+    return {
+      usdcPurchase: input.usdc_purchase
+    }
+  } else if ('tip_user_id' in input) {
+    return {
+      tipUserId: encodeHashId(input.tip_user_id)
+    }
+  } else {
+    throw new Error(`Unsupported access conditions type: ${input}`)
```

Comment:
fwiw I think this is going to log `[Object object]` for `input`? Worth checking.